README.md

```diff
@@ -1,2 +1,51 @@
 # eigen3_cmake_module
-Adds a custom find module for Eigen3
+
+This package adds a [CMake find module](https://cmake.org/cmake/help/v3.14/manual/cmake-developer.7.html#find-modulesjj) for [Eigen3](https://eigen.tuxfamily.org/dox/) that sets [standard CMake variables](https://cmake.org/cmake/help/v3.5/manual/cmake-developer.7.html#standard-variable-names).
+This enables `ament_export_dependencies(Eigen3)` and `ament_target_dependencies(my_target Eigen3)`.
```

Comment:
Is it possible to give a little more context here on this issue? In your explanation you mentioned the reasoning behind this package is that ament process for adding eigen3 as a dependency may not work in Ubuntu bionic. Would the user be able to recognize that that's the problem when their "library or executable using Eigen3" failed to compile? Would someone searching for a fix be able to recognize that this is for that exact issue? 

For your audience to be convinced that this is the solution to their problem, I just want to make sure that the "solution" mentioned here is enough to do so. 

Maybe "This enables ... _in the event that_ ... (it fails because x)"

Reply:
I added more context about the problem in e78118d in #2